app/component/ItinerarySummaryListContainer/components/errorCardProperties.js

```diff
@@ -0,0 +1,205 @@
+import PastLink from './components/PastLink';
+import ChangeDepartureTimeLink from './components/ChangeDepartureTimeLink';
+
+const info = { iconType: ICON_TYPE_INFO, iconImg: ICON_INFO };
+const caution = { iconImg: ICON_CAUTION, iconType: ICON_TYPE_CAUTION };
+
+/**
+ * Priority-ordered list of properties to describe error card rendering.
+ * If many error messages match the user's query, first in the 'errorCardProps'
+ * should be shown to user.
+ */
+const errorCardProps = [
+  {
+    id: 'no-route-already-at-destination',
+    props: {
+      bodyId: 'no-route-already-at-destination',
+      LinkComponent: null,
+      ...info,
+    },
+  },
+  {
+    id: 'no-route-origin-same-as-destination',
+    props: {
+      bodyId: 'no-route-origin-same-as-destination',
+      ...info,
+    },
+  },
+  {
+    id: 'itinerary-in-the-past',
+    props: {
+      ...info,
+      titleId: 'itinerary-in-the-past-title',
+      bodyId: 'itinerary-in-the-past',
+      LinkComponent: PastLink,
+    },
+  },
+  {
+    id: 'outside-service-period',
+    props: {
+      titleId: 'router-unable',
+      bodyId: 'router-outside-service-period',
+      LinkComponent: PastLink,
+      ...info,
+    },
+  },
+  {
+    id: 'no-transit-connection',
+    props: {
+      titleId: 'no-route-msg',
+      bodyId: 'router-transit-connection',
+      ...info,
+    },
+  },
+  {
+    id: 'outside-bounds-1',
+    props: {
+      titleId: 'router-unable',
+      bodyId: 'destination-outside-service',
+      LinkComponent: NationalServiceLink,
+      ...info,
+    },
+  },
+  {
+    id: 'outside-bounds-2',
+    props: {
+      titleId: 'router-unable',
+      bodyId: 'origin-outside-service',
+      LinkComponent: NationalServiceLink,
+      ...info,
+    },
+  },
+  {
+    id: 'outside-bounds-3',
+    props: {
+      titleId: 'router-unable',
+      bodyId: 'router-outside-bounds-3',
+      LinkComponent: NationalServiceLink,
+      ...info,
+    },
+  },
+  {
+    id: 'location-not-found-1',
+    props: {
+      titleId: 'router-location-not-found-title-1',
+      bodyId: 'router-location-not-found',
+      ...info,
+    },
+  },
+  {
+    id: 'location-not-found-2',
+    props: {
+      titleId: 'router-location-not-found-title-2',
+      bodyId: 'router-location-not-found',
+      ...info,
+    },
+  },
+  {
+    id: 'location-not-found-3',
+    props: {
+      titleId: 'router-location-not-found-title-3',
+      bodyId: 'router-location-not-found',
+      ...info,
+    },
+  },
+  {
+    id: 'no-stops-in-range-1',
+    props: {
+      titleId: 'router-unable',
+      bodyId: 'router-stops-in-range-1',
+      ...info,
+    },
+  },
+  {
+    id: 'no-stops-in-range-2',
+    props: {
+      titleId: 'router-unable',
+      bodyId: 'router-stops-in-range-2',
+      ...info,
+    },
+  },
+  {
+    id: 'no-stops-in-range-3',
+    props: {
+      titleId: 'router-unable',
+      bodyId: 'router-stops-in-range-3',
+      ...info,
+    },
+  },
+  {
+    id: 'walk-bike-itinerary-4',
```

Comment:
why is 4 before 1 in this list?

Reply:
This directly from priority definition: https://github.com/HSLdevcom/digitransit-ui/blob/v2/app/component/ItinerarySummaryListContainer.js#L263